src/InstallationValidator.Core/Services/BatchOutputLoader.cs

```diff
@@ -11,15 +14,95 @@ public interface IBatchOutputLoader
 
    public class BatchOutputLoader : IBatchOutputLoader
    {
+      private readonly IDimensionFactory _dimensionFactory;
+
+      public BatchOutputLoader(IDimensionFactory dimensionFactory)
+      {
+         _dimensionFactory = dimensionFactory;
+      }
+
       public BatchSimulationExport LoadFrom(string fileName)
       {
          var serializer = new JsonSerializer();
 
-         using (var sr = new StreamReader(fileName))
-         using (var reader = new JsonTextReader(sr))
+         try
+         {
+            using (var sr = new StreamReader(fileName))
+            using (var reader = new JsonTextReader(sr))
+            {
+               return serializer.Deserialize<BatchSimulationExport>(reader);
+            }
+         }
+         catch (Exception e)
+         {
+            //try to load the old way and convert to new way. Temporary for v11
+            //required to load again since the reader was consumed
+            using (var sr = new StreamReader(fileName))
+            using (var reader = new JsonTextReader(sr))
+            {
+               var oldBatchSimulationExport = serializer.Deserialize<OldBatchSimulationExport>(reader);
```

Comment:
THis can be removed when we release v12. Right now this is the only simple way to read the old structure